src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String FIRST_PARAMETER = "supply";
+    private static final String SECOND_PARAMETER = "buy";
+    private static final String THIRD_PARAMETER = "result";
+    private static final String SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String stringsFromFile = readFromFile(fromFileName);
+        String report = createReport(stringsFromFile);
+        writeReport(report, toFileName);
+    }
+
+    private String readFromFile(String fileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName));
+
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(';');
```

Comment:
You can try to use LIst instead of the Builder, to avoid splitting
```suggestion
                result.add(value;)
```

Reply:
Also, you can try to use Filed.readAllLines method

Reply:
Wanted to do so at first, but were broblems. Do not understand, where is a mistake. Help please!
https://prnt.sc/Iu-qpp9CYuW2